build-release.sh

```diff
@@ -360,15 +362,15 @@ if [ "${build_classical}" == "1" ]; then
   cp -r deps/vulkansdk-macos/MoltenVK/MoltenVK.xcframework ios_xcode/
   rm -rf ios_xcode/MoltenVK.xcframework/{macos,tvos}*
   cd ios_xcode
-  zip -q -9 -r "${templatesdir}/ios.zip" *
+  $ZIP -r "${templatesdir}/ios.zip" *
   cd ..
   rm -rf ios_xcode
 
   ## Templates TPZ (Classical) ##
 
   echo "${templates_version}" > ${templatesdir}/version.txt
   pushd ${templatesdir}/..
-  zip -q -9 -r -D "${reldir}/${godot_basename}_export_templates.tpz" templates/*
+  $ZIP -r "${reldir}/${godot_basename}_export_templates.tpz" templates/*
```

Comment:
I think `-D` was important here, so this might break the logic.

Reply:
`7z` doesn't seem to have an equivalent of `-D`. According to `zip`'s command line help:

```
 -D   do not add directory entries
```

The only difference I can notice between 2 ZIP archives created (one with `-D`, one without `-D`) is that the former will have directory entries in its `7z l` listing, but the latter will specify full paths to every file.

In practice, this appears to make no concrete difference on what you see when opening the ZIP archive: both have the exact same folder structure when viewed with GUI archive managers. In short, `-D` does **not** make the ZIP's structure flat or remove the top-level folder as the option's name might suggest.

I think the `-D` requirement is a leftover from early Godot 3.x days where Godot behaved poorly if given a TPZ that had directory entries.